invenio_rdm_records/ext.py

```diff
@@ -24,6 +24,7 @@
 from .services import RDMFileDraftServiceConfig, RDMFileRecordServiceConfig, \
     RDMRecordService, RDMRecordServiceConfig
 from .services.schemas.metadata_extensions import MetadataExtensions
+from .services.secret_links import SecretLinkService
```

Comment:
moderate: it should be possible to import from "services", by exposing it in the corresponding `__init__.py`.

```suggestion
from .services import RDMFileDraftServiceConfig, RDMFileRecordServiceConfig, \
    RDMRecordService, RDMRecordServiceConfig, SecretLinkService
from .services.schemas.metadata_extensions import MetadataExtensions
```